### PR TITLE
Main update for node 3.7.0 compatibility

### PR DESCRIPTION
**Pull request by frankTheTank72**
You need minimum Signum-Node 3.7.0 to run this upgrade for the explorer

### CODE REVIEW

File: scan/templates/accounts/subscription.html
Comment (pir8radio) [inline scan/templates/accounts/subscription.html:1]: the TD's need to have the same class as the TH or when you resize to mobile the title will shrink down but the table will not.  can not display all info on mobile you must pick important info only, you should not scroll left and right on mobile devices. 

image

File: scan/templates/accounts/alias.html
Comment (pir8radio) [inline scan/templates/accounts/alias.html:1]: Same issue here,   you need to class the columns so only important items show in mobile view..  so you do not need to scroll left and right..      only show important info on mobile, until you click TX.

image

File: scan/templates/assets/transfers_list.html
Comment (pir8radio) [inline scan/templates/assets/transfers_list.html:1]: you un-hide the ticker and token id in the table view?      why?  now there is wasted space in table, same ticker info is in the header, redundant info.

image

File: scan/templates/accounts/alias.html
Comment (pir8radio) [inline scan/templates/accounts/alias.html:1]: consider hiding TX ID, BLOCK, and LAST MODIFIED for mobile view...   Also why show OWNER in the account alias view?  we know who the owner is, we are viewing in his account tab...

image

**Review by pir8radio — COMMENTED** [review]
Comments added

File: scan/templates/accounts/subscription.html
Comment (frankTheTank72) [reply scan/templates/accounts/subscription.html:1]: okay fixed this

Comment (frankTheTank72) [reply scan/templates/accounts/subscription.html:1]: also fixed

File: scan/templates/accounts/alias.html
Comment (frankTheTank72) [reply scan/templates/accounts/alias.html:1]: also fixed

Comment (frankTheTank72) [reply scan/templates/accounts/alias.html:1]: it is a shared view  that is  why we see now the owner here - maybe a update for later

File: scan/templates/assets/transfers_list.html
Comment (frankTheTank72) [reply scan/templates/assets/transfers_list.html:1]: yes i changed it back - good catch

**Pull request merged**